complete/src/main/java/com/example/restservice/service/smsService/MockSmsService.java

```diff
@@ -1,13 +1,18 @@
 package com.example.restservice.service.smsService;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.context.annotation.Profile;
 import org.springframework.stereotype.Component;
 
 @Component
 @Profile("!prod")
 public class MockSmsService implements SmsService {
+    private final Logger LOGGER = LoggerFactory.getLogger(MockSmsService.class);
```

Comment:
lombok has a slf4j annotation, can you use that?
https://projectlombok.org/api/lombok/extern/slf4j/Slf4j.html